ui/ui/config_ui.py

```diff
@@ -18,9 +19,31 @@ def show_page_content(self):
 
         st.markdown(f"""
                     # ARISE configuration landing page
-                    
-                    ## The UI configuration is:  
                     """)
+        st.divider
+        # allows the user to configure the base path for the prediction model
+        with st.form(key='config-form'):
+            st.subheader("Configure the base path of the prediction model")
```

Comment:
please change the title of the page from "Configure the base path of the prediction model" to 
"Configure the input path of the prediction model"

Reply:
done